app/Services/ForwardChatworkService.php

```diff
@@ -117,4 +130,52 @@ function ($match) use ($params, &$isMatching) {
 
         return '';
     }
+
+    public function sendMessages($messages, $payload_history_id)
+    {
+        $bot = $this->webhook->bot;
+        $room_id = $this->webhook->room_id;
+        $chatwork = Chatwork::withAPIToken($bot->bot_key);
+
+        foreach($messages as $key=>$message) {
+            try {
+                $chatwork->room($room_id)->messages()->create($message);
+                array_splice($messages, $key, 1);
+                // save success message_history
+                $this->saveMessageHistory($payload_history_id, $message, MessageHistoryStatus::SUCCESS);
+            } catch (APIException $error) {
+                switch ($error->getStatus()) {
+                    case 403:
+                        // permission denined
+                        $this->saveMessageHistory($payload_history_id, $message, MessageHistoryStatus::FAILED, 'Permission');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Services/ForwardChatworkService.php</br>**Line**: 150</br>Variable "payload_history_id" is not in valid camel caps format